lib/ansible/modules/windows/win_nssm.ps1

```diff
@@ -96,9 +96,10 @@ Function Nssm-Remove
 
     if (Service-Exists -name $name)
     {
-        $cmd = "stop ""$name"""
-        $results = Nssm-Invoke $cmd
-
+        if ((get-service $name).Status -ne "Stopped") {
```

Comment:
suggest 
if (( get-service "$name").Status -ne "Stopped") {

(just in case service name contains spaces).